src/ReservoirPair.sol

```diff
@@ -512,13 +514,18 @@ abstract contract ReservoirPair is IAssetManagedPair, ReservoirERC20 {
             return (aCurrRawPrice, int112(LogCompression.toLowResLog(aCurrRawPrice)));
         }
 
+        // call to `percentDelta` is safe as long as the difference between aCurrRawPrice and aPrevClampedPrice is
+        // less than uint196 (1e59). It is extremely unlikely that one trade can change the price by 1e59
         if (aCurrRawPrice.percentDelta(aPrevClampedPrice) > maxChangeRate * aTimeElapsed) {
```

Comment:
on a technical level, it is possible to overflow this with a very large and a very small number. But I think it's so unlikely that I'm willing to let this one swap revert if this case happens

Reply:
You need to say that if this happens it will revert. You will make people think we can be hacked with this comment

Reply:
updated comment to say that it will revert when the inputs are beyond a certain range